### DISCUSSION TITLE
Support snapcraft layout

### DISCUSSION THREAD

**Discussion by PeerXu** [goreleaser/goreleaser#2109] [Q&A] [answered]
Goreleaser is a great project to help me to build my project.

It support many difference pipe, like snapcraft.

But when I am using snapcraft pipe, `layout` feature is missing in snapcraft pipe.

Then I try to patch some code to support `layout` feature.

Finally, I am implemented the feature, and test pass in local environment(`go test ./internal/pipe/snapcraft`),

but failed to test in github workflow build.

Is any worth with my testcase?

[error log](https://github.com/PeerXu/goreleaser/runs/2110323236?check_suite_focus=true)

[edited]

A few hour later...

PR close and reopen, the test is passed.

What happened?

[test log](https://github.com/PeerXu/goreleaser/runs/2110758333?check_suite_focus=true)

**Comment by caarlos0** [comment, marked as answer]
This is now available https://goreleaser.com/customization/snapcraft/

**Reply by PeerXu** [reply]
thx!
